src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = FigureSupplier.getRandomFigure();
+        }
+
+        for (int i = figures.length / 2; i < figures.length; i++) {
+            figures[i] = FigureSupplier.getDefaultFigure();
```

Comment:
Similarly, instead of calling `FigureSupplier.getDefaultFigure()` statically, use the `figureSupplier` instance created on line 6 to call `getDefaultFigure()`.